### DISCUSSION TITLE
Doris-manager 中 停止服务的命令是不是可以调整一下？

### DISCUSSION THREAD

**Discussion by boneHCL** [apache/doris#9163] [RFC]
如下图所示，Doris-manager中的stop_manager.sh脚本命令：ps -ef|grep doris-manager|grep -v grep|cut -c 9-15|xargs kill -9

![1650595398](https://user-images.githubusercontent.com/76100619/164586657-02328094-526a-41fa-94f0-c02bb9d34e46.jpg)

如果服务器长时间为重启，启动服务的进程id就会超过 9-15的范围，使用脚本停止服务就是失败。

![image](https://user-images.githubusercontent.com/76100619/164586954-7e9a6673-155a-42c3-9af7-73b6aa282837.png)

![image](https://user-images.githubusercontent.com/76100619/164587004-7ef84bf0-f9f6-45f3-b8ae-a1bbb6792083.png)

能不能启动的时候在bin目录下保存pid文件，下次停止服务的时候读取pid文件，关闭服务并删除pid文件？

**Comment by JNSimba** [comment]
Good idea, are you interested in making a pr?

**Comment by KeithTt** [comment]
@JNSimba I create a PR to update this. 

https://github.com/apache/doris-manager/pull/64
